sphinx_gallery/gen_rst.py

```diff
@@ -124,7 +125,7 @@ def flush(self):
 """
 
 
-CODE_OUTPUT = """.. rst-class:: sphx-glr-script-out
+CODE_OUTPUT = u""".. rst-class:: sphx-glr-script-out
```

Comment:
I'm not quite sure what you mean. You mean in the modules you work on that
use sphinx-gallery you've started putting in a bunch of those lines? Or
you've started putting them in the sphinx-gallery code itself?

So far we haven't needed to use them to have unicode work properly. Does it
make them work better with sphinx-gallery, or is there some other
advantage?